core/modules/search/search.module

```diff
@@ -975,13 +975,26 @@ function search_form($form, &$form_state, $action = '', $keys = '', $module = NU
  * @see search_box_form_submit()
  */
 function search_box($form, &$form_state, $form_id) {
+  $search_field_placeholder = config_get('search.settings', 'search_field_placeholder');
+  if (!empty($search_field_placeholder)) {
+    $attributes = array(
```

Comment:
`$attributes` array should be initialized before to avoid repeating the `'title' => t('Enter the terms you wish to search for.'),` part.

Reply:
Thanks, I'll fix it from home (after few hours).